src/api/types/playbooks.ts

```diff
@@ -61,10 +61,17 @@ export interface PlaybookRun extends CreatedAt, Avatar, Agent {
   config?: Pick<ConfigItem, "id" | "name" | "type" | "config_class">;
 }
 
-type PlaybookSpecFilterType = {
-  type?: string;
-  tags?: string[];
-  labelSelector?: string[];
+export type PlaybookResourceSelector = {
```

Comment:
This should move to search service as this api. is not only for use by playbooks